aiohttp tests fail if repo dir isn't named "connexion"

### Description
if you clone the repo under a different name, the aiohttp tests will fail.

### Expected behaviour
Tests should be agnostic to the name of the directory that the repo was cloned into.

### Actual behaviour
tests/aiohttp/test_aiohttp_app.py :: test_app_get_root_path
fails with
```
========================================================== FAILURES ===========================================================
___________________________________________________ test_app_get_root_path ____________________________________________________

aiohttp_api_spec_dir = PosixPath('/home/daniel/stash/connexion-other-name/tests/fixtures/aiohttp')

    def test_app_get_root_path(aiohttp_api_spec_dir):
        app = AioHttpApp(__name__, port=5001,
                         specification_dir=aiohttp_api_spec_dir)
>       assert app.get_root_path().endswith(os.path.join('connexion', 'tests', 'aiohttp')) == True
E       AssertionError: assert False == True
E        +  where False = <built-in method endswith of str object at 0x7faaf325e2a0>('connexion/tests/aiohttp')
E        +    where <built-in method endswith of str object at 0x7faaf325e2a0> = '/home/daniel/stash/connexion-other-name/tests/aiohttp'.endswith
E        +      where '/home/daniel/stash/connexion-other-name/tests/aiohttp' = <bound method AioHttpApp.get_root_path of <connexion.apps.aiohttp_app.AioHttpApp object at 0x7faaf2d064e0>>()
E        +        where <bound method AioHttpApp.get_root_path of <connexion.apps.aiohttp_app.AioHttpApp object at 0x7faaf2d064e0>> = <connexion.apps.aiohttp_app.AioHttpApp object at 0x7faaf2d064e0>.get_root_path
E        +    and   'connexion/tests/aiohttp' = <function join at 0x7faaf5944a60>('connexion', 'tests', 'aiohttp')
E        +      where <function join at 0x7faaf5944a60> = <module 'posixpath' from '/home/daniel/stash/connexion-other-name/venv/lib/python3.6/posixpath.py'>.join
E        +        where <module 'posixpath' from '/home/daniel/stash/connexion-other-name/venv/lib/python3.6/posixpath.py'> = os.path

tests/aiohttp/test_aiohttp_app.py:78: AssertionError
```

### Steps to reproduce
git clone https://github.com/zalando/connexion.git connexion-other-name
cd connexion-other-name
# setup tests...
pytest

### Additional info:
python version 3.6
connexion version d18c387

--- a/tests/aiohttp/test_aiohttp_app.py
+++ b/tests/aiohttp/test_aiohttp_app.py
@@ -75,7 +75,7 @@ def test_app_run_server_error(web_run_app_mock, aiohttp_api_spec_dir):
 def test_app_get_root_path(aiohttp_api_spec_dir):
     app = AioHttpApp(__name__, port=5001,
                      specification_dir=aiohttp_api_spec_dir)
-    assert app.get_root_path().endswith(os.path.join('connexion', 'tests', 'aiohttp')) == True
+    assert app.get_root_path().endswith(os.path.join('tests', 'aiohttp')) == True
 
 
 def test_app_get_root_path_not_in_sys_modules(sys_modules_mock, aiohttp_api_spec_dir):

